src/app/scripts/cac/control/cac-control-explore.js

```diff
@@ -401,6 +415,8 @@ CAC.Control.Explore = (function (_, $, Geocoder, MapTemplates, HomeTemplates, Ro
 
         // now places list has been updated, go fetch the travel time
         // from the new origin to each place
+
+        // TODO: cache travel times for last origin/routing params set?
```

Comment:
Is there an issue about this?

Reply:
Not yet, but it occurred to me that with the new filters, requeries will be fired off more frequently and often times will be unnecessary. I'll go make an issue and note it here.

Reply:
#945